fun/templates/fun/emails/base_email.html

```diff
@@ -19,12 +19,6 @@
 <body>
 <table style="width: 90%; margin: auto; border: 8px solid white; border-collapse:collapse;">
 
-<tr>
-    <td width="100%" class="logo" style="text-align: center; background-color: #EFECDD; padding: 12px;">
-        <img src="{{ STATIC_URL }}selftest/images/logo_fun.png" alt="FUN">
-    </td>
-</tr>
```

Comment:
isn't this going to break some emails? Should we keep this in the email base and move the image to another app?

Reply:
No, this email template is not used anymore, and `{{ STATIC_URL }}` would create a relative link, which can't work in an email.